ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -336,9 +336,10 @@ export default defineComponent({
       localState.searched = true
       localState.searchType = getSearchedType.value?.searchTypeUI || ''
       localState.results = sortByStatus(localState.activeResults)
-      if (!props.isReviewMode) {
-        localState.results = localState.results.map(result => ({ ...result, includeLienInfo: false }))
-      }
+      localState.results = localState.results.map(result => {
+        // includeLienInfo needs to be initialized or something weird will happen
+        return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
+      })
```

Comment:
So you want to do this for both Results selection and Results selection review modes?

Reply:
it doesn't really make any difference in terms of UX behavior, just... less code.

Reply:
Ok sounds good, if it was intentional!

Resolved